src/modules/navigation/duck/reducer.js

```diff
@@ -14,8 +18,21 @@ const sort = (state = null, action) => {
   }
 }
 
+// Reducer for the redirection flag
+const operationRedirectedReducer = (state = false, action) => {
+  switch (action.type) {
+    case OPERATION_REDIRECTED:
+      return true
+    case RESET_OPERATION_REDIRECTED:
+      return false
+    default:
+      return state
+  }
+}
+
 export default combineReducers({
-  sort
+  sort,
+  operationRedirected: operationRedirectedReducer // Add the new reducer
```

Comment:
We should probably change this comment, the new state is new now, but won't be tomorrow